CSharpMath.Editor/MathKeyboard.cs

```diff
@@ -62,7 +64,7 @@ public class MathKeyboard<TFont, TGlyph> where TFont : IFont<TGlyph> {
         blinkTimer.Stop();
         blinkTimer.Start();
         if (value != MathKeyboardCaretState.Hidden &&
-           MathList.AtomAt(_insertionIndex) is Atoms.Placeholder placeholder) 
```

Comment:
I did not see this change when I submitted and still don't see this change in the diff in my local environment. I don't know what this is.